.gitlab-ci.yml

```diff
@@ -3844,13 +3818,13 @@ latest_release_7:
 #
 .deploy_cloudfront_invalidate: &deploy_cloudfront_invalidate
   stage: deploy_invalidate
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/deploy:$DATADOG_AGENT_BUILDERS
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/gitlab_agent_deploy:$DATADOG_AGENT_BUILDERS
   tags: [ "runner:main", "size:large" ]
   dependencies: []
   script:
-    - cd /deploy_scripts/cloudfront-invalidation
-    - "REPO=apt PATTERN_SUBSTRING=/$DEB_RPM_BUCKET_BRANCH/ ./invalidate.sh"
-    - "REPO=yum PATTERN_SUBSTRING=/$DEB_RPM_BUCKET_BRANCH/ ./invalidate.sh"
+    - cd ~/deploy_scripts/cloudfront-invalidation
+    - "REPO=apt REPO_ENV=staging PATTERN_SUBSTRING=/$DEB_RPM_BUCKET_BRANCH/ ./invalidate.sh"
```

Comment:
Was staging the default before?

Reply:
Indeed. The old `invalidate.sh`/`invalidate.rb` scripts were only able to send artifacts to the staging repositories.